rules/symfony/src/Rector/ClassMethod/ConsoleExecuteReturnIntRector.php

```diff
@@ -112,6 +112,11 @@ private function addReturn0ToMethod(ClassMethod $classMethod): void
                 return null;
             }
 
+            if ($node->expr instanceof Int_) {
+                $hasReturn = true;
```

Comment:
I think this value is not need, since it's used bellow to return `$node`/null.

What do you think?

Reply:
yes, updated.